tests/e2e/pageobjects/ide/plugins/GitHubPullRequestPlugin.ts

```diff
@@ -25,7 +25,7 @@ export class GitHubPullRequestPlugin {
     constructor(@inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
         @inject(CLASSES.LeftToolBar) private readonly leftToolbar: LeftToolBar,
         @inject(CLASSES.TopMenu) private readonly topMenu: TopMenu,
-        @inject(CLASSES.QuickOpenContainer) private readonly quickOpenContainer: QuickOpenContainer
+        @inject(CLASSES.QuickOpenContainer) private readonly quickOpenContainer: QuickOpenContainer,
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.

Reply:
fixed with the linter code change